lib/deface/railtie.rb

```diff
@@ -95,7 +95,7 @@ def tweak_eager_loading(railtie)
       paths_to_reject = railtie.config.eager_load_paths.select { |path| path.to_s  =~ /app\/overrides\z/ }
       railtie.config.eager_load_paths = railtie.config.eager_load_paths.reject { |path| path.in?(paths_to_reject) }
 
-      if Rails.version >= "6.0" && Rails.configuration.autoloader == :zeitwerk
+      if !Deface.before_6? && Rails.configuration.autoloader == :zeitwerk
```

Comment:
Should we have `Deface.rails_6_or_above?`? It takes me a second to process the meaning of `!Deface.before_6?`.